app/src/main/java/com/door43/translationstudio/core/Translator.java

```diff
@@ -322,7 +323,10 @@ public TargetTranslation importDraftTranslation(NativeSpeaker translator, Source
             FileUtils.deleteQuietly(tempCache);
         }
 
-        return importedTargetTranslationSlugs.toArray(new String[importedTargetTranslationSlugs.size()]);
+        String[] translationSlugs = importedTargetTranslationSlugs.toArray(new String[importedTargetTranslationSlugs.size()]);
+        boolean success = TargetTranslationMigrator.migrate(tempCache,translationSlugs);
```

Comment:
This is redundant since we have already ran the migration by the time we reach this line.